### PR TITLE
made context switch course - files more usable

### PR DESCRIPTION
**Pull request by dominikbruechner**
+ added footer margin in mobile
+ made context switch course - files more usable
screenshot 2018-12-05 at 16 10 01

screenshot 2018-12-05 at 16 10 27

screenshot 2018-12-05 at 16 13 54

### CODE REVIEW

File: static/styles/lib/base.scss [hunk 1]
```diff
@@ -160,6 +160,7 @@ footer {
 
 		@include media-breakpoint-down(xs) {
 			background: $colorBeige;
+			margin-top: 50px;
```
Comment (janrenz) [inline static/styles/lib/base.scss]: this coudl be moved to a constant

Comment (dominikbruechner) [reply static/styles/lib/base.scss]: like $margin-top-50 ?

Comment (janrenz) [reply static/styles/lib/base.scss]: more than related to the grid system margin-top: $2gridUnits

Comment (adrianjost) [reply static/styles/lib/base.scss]: Why? It's that specific here. 
In addition, there is already bootstrap stuff like `mt-1`, `mt-2`, ...

Comment (schurigh) [reply static/styles/lib/base.scss]: I would not do that, constant for a specific margin-top value? nah...

Comment (janrenz) [reply static/styles/lib/base.scss]: If we follow a global grid there are no specific margins. But its just a proposal, can easily changed later. and maybe something that we need to discuss with mona

**Pull request merged**